src/index.html

```diff
@@ -2,16 +2,139 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080" method="post" 
+    class="form">
+      <div class="form__block block">
+        <fieldset class="block-title">
+          <legend class="block__subtitle">Personal information</legend>
+          <div class="block__item item-block">
+            <label for="surname" class="item-block__label">Surname:
+              <input id="surname" autocomplete="off" 
+              type="text" class="item-block__input">
```

Comment:
when more than 2 props => each prop goes to new line, fix everywhere

```suggestion
              <input
                id="surname"
                autocomplete="off"
                type="text"
                class="item-block__input"
              >
```